attributes/plugins.rb

```diff
@@ -28,12 +28,12 @@
   docker-workflow:1.10
   durable-task:1.17
   external-monitor-job:1.4
-  ghprb:1.40.0
-  git:3.8.0
+  ghprb:1.42.0
```

Comment:
Do we want this plugin in the array in the `ibmz` and `powerci` recipes?

Reply:
It's not needed there.